### DISCUSSION TITLE
Allows creation of duplicates

### DISCUSSION THREAD

**Discussion by edalzell** [joshhanley/livewire-autocomplete#41] [Help] [answered]
Right now, it's quite easy to accidentally create duplicates:

![image](https://github.com/user-attachments/assets/528a137d-36c9-40d5-bc56-885c4edb8849)

Is there a nice way to handle this?

**Comment by joshhanley** [comment, marked as answer]
Sorted on Discord.

TL;DR: It's just blade, so you can wrap the `x-autocomplete-new-item` component in a conditional to hide it if there is a direct match.

Or pass the conditional into the `show` attribute on the component.
